feasto_core_clean_arch/storages/storage_implementation.py

```diff
@@ -0,0 +1,344 @@
+
+    def get_item(self, item_id: int, restaurant_id: int) -> ItemDTO:
+        item = Item.objects.get(id=item_id)
+
+        item_dto = ItemDTO(
+            name=item.name,
+            id=item.id,
+            available_quantity=item.available_quantity,
+            restaurant_id=item.restaurant_id
+        )
+        return item_dto
+
+    def add_item(self,
+                    restaurant_id: int, name: str = None, available_quantity: int = None) -> ItemDTO:
+        item = Item.objects.create(
+            name=name,
+            available_quantity=available_quantity,
+            restaurant_id=restaurant_id,
+        )
+
+        item_dto = ItemDTO(
+            name= item.name,
+            id= item.id,
+            available_quantity= item.available_quantity,
+            restaurant_id= item.restaurant_id
+        )
+
+        return item_dto
+
+    def add_restaurant(self,
+                 name: str, user_id: str, status: StatusType, location: str) -> RestaurantDTO:
+        user = User.objects.get(id = user_id)
+        restaurant = Restaurant.objects.create(
+            name=name,
+            user=user,
+            location=location,
+            status=status
+        )
+
+        user_dto = self.get_user(user_id)
+        restaurant_dto = RestaurantDTO(
+            id= restaurant.id,
+            name= restaurant.name,
+            user= user_dto,
+            location=restaurant.location,
+            status= restaurant.status
+
+        )
+
+        return restaurant_dto
+
+
+    def update_item(self, item_id: int, restaurant_id: int,user_id: int,
+                name: str= None, available_quantity:int= None) -> ItemDTO:
+        item = Item.objects.get(id=item_id)
+
+        if name:
+            item.name = name
+
+        if available_quantity:
+            item.available_quantity = available_quantity
+
+        item.save()
+
+        item_dto = ItemDTO(
+            id=item.id,
+            name= item.name,
+            available_quantity=item.available_quantity,
+            restaurant_id=item.restaurant_id
+        )
+
+        return item_dto
+
+    def update_restaurant(self,
+                    name: str, status: StatusType, user_id: str, rest_id: int ) -> RestaurantDTO:
+        updated_restaurant = Restaurant.objects.get(id=rest_id)
+
+        if name:
+            updated_restaurant.name = name
+
+        if status:
+            updated_restaurant.status = status
+
+        updated_restaurant.save()
+        user_dto = self.get_user(user_id)
+        restaurant_data = RestaurantDTO(
+            id= updated_restaurant.id,
+            name= updated_restaurant.name,
+            user= user_dto,
+            location= updated_restaurant.location,
+            status= updated_restaurant.status
+        )
+
+        return restaurant_data
+
+    def delete_restaurant(self,
+                          restaurant_id: int, user_id: int):
+        rest = Restaurant.objects.get(id = restaurant_id)
```

Comment:
in python, we won't have the space in calling args..
i.e, id = restaurant_id shall be `id=restaurant_id`